docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
 ---
-title: "User administration"
+title: "Users"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a role that controls their access and permission levels.
```

Comment:
I'm not sure that this provides a complete description of this page without context. It should include the idea that this page shows readers how to manage the roles and maybe about adding and deleting them?

Reply:
What I mean is that you mention I took that part out but it is there, or perhaps you want to see the tabs in a different manner?
Are you referring to the structure itself? something like this?
## Managing users from the console
### Add a user to a project
### Delete a user from a project
## Managing users with the CLI

I'm happy to jump on a call and clarify.

Reply:
Ok thanks